### PR TITLE
Feature/remove parse

### PR DESCRIPTION
**Pull request by ryanrampage1**
When you see the retrofit and ok http dependencies, it is a good thing not bad. They split the library into smaller components so it doesnt have such a large dex impact unless you need stuff.

### CODE REVIEW

File: app/src/main/java/com/spartahack/spartahack17/Retrofit/ISpartahackAPIService.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import retrofit2.http.DELETE;
+import retrofit2.http.POST;
+import retrofit2.http.Path;
+import rx.Observable;
+
+/**
+ * Created by ryancasler on 9/19/16.
+ * SpartaHack2016-Android
+ */
+public interface ISpartahackAPIService {
```
Comment (mhammerly) [inline app/src/main/java/com/spartahack/spartahack17/Retrofit/ISpartahackAPIService.java]: ISparta**H**ackAPIService\* ;)

**Review by mhammerly — COMMENTED** [review]
will leave actual review to @memuyskens

File: app/src/main/java/com/spartahack/spartahack17/Retrofit/ISpartahackAPIService.java (hunk 1 above)
Comment (mhammerly) [reply app/src/main/java/com/spartahack/spartahack17/Retrofit/ISpartahackAPIService.java]: also looks like the repo name is still SpartaHack2016-Android in your IDE settings haha

**Review by memuyskens — COMMENTED** [review]
Is the token hardcoded in SpartahackAPIService.java?

**Comment by ryanrampage1** [comment]
i moved it @memuyskens

File: app/src/main/java/com/spartahack/spartahack17/Retrofit/ISpartahackAPIService.java (hunk 1 above)
Comment (ryanrampage1) [reply app/src/main/java/com/spartahack/spartahack17/Retrofit/ISpartahackAPIService.java]: @mhammerly my next pr is going to be removing all the android studio stuff and adding it to git ignore

**Review by memuyskens — APPROVED** [review]

**Pull request merged**